src/shared/backend/graph/flash/index.ts

```diff
@@ -125,7 +142,20 @@ builder.mutationType({
       resolve: async (_, { firmware, deviceId }, context) => {
         let firmwareData: Buffer | undefined;
         let firmwareBundleUrl: string | undefined;
-        if (firmware.version === "local") {
+        let fetchCloudbuild = false;
```

Comment:
This could probably just be `cloudBuild`?

Reply:
Same as later, maybe `isCloudBuild` to not conflict with cloudbuild (methods)

Reply:
isCloudBuild is better :)